vpr/src/route/rr_graph_storage.h

```diff
@@ -72,6 +75,11 @@ struct t_rr_node_ptc_data {
 static_assert(sizeof(t_rr_node_data) == 16, "Check t_rr_node_data size");
 static_assert(alignof(t_rr_node_data) == 16, "Check t_rr_node_data size");
 
+// aligned_allocator is a STL allocator that allocates memory in an aligned
+// fashion (if supported by the platform).
+//
+// It is worth noting the C++20 std::allocator does aligned allocations, but
+// C++20 has poor support.
 template<class T>
 struct aligned_allocator {
```

Comment:
It seems like this allocator should probably be in vtr_memory.h

Reply:
Yep